build/configs/imxrt1020-evk/imxrt1020-evk_download.sh

```diff
@@ -98,6 +98,7 @@ function get_executable_name()
 		app) echo "tinyara_user.bin";;
 		micom) echo "micom";;
 		wifi) echo "wifi";;
+		userfs) echo "imxrt1020-evk_smartfs.bin";;
```

Comment:
Corresponding change in get_partition_index is also needed.

Reply:
@btheosam, partition index for flat and protection build would be different, how to we handle this case?

Reply:
How about changing #Download <list of partitions> to check if userfs and download to last partition always.

Reply:
@sijupunnnoose 
get the index from 'parts' variable and echo back.

Reply:
@btheosam @sijupunnnoose As you said, it should get the partition number by parsing partition config. It should not be hard-coded. I think it can be indexed in for loop.

Reply:
@sijupunnnoose @btheosam @sunghan-chang , I have modified the logic. Please review.